examples/bcs_mean_field.py

```diff
@@ -158,7 +158,7 @@ def main():
     print('Quantum circuits to prepare the BCS meanfield state.')
     print('Number of sites = ', n_site)
     print('Number of fermions = ', n_fermi)
-    print('Tunneling strength = ', t)
```

Comment:
This is correct in [American English](https://sapling.ai/usage/tunneling-vs-tunnelling)